From: =?utf-8?b?T25kxZllaiBTdXLDvQ==?= <ondrej@debian.org>
Date: Wed, 21 Oct 2015 10:46:05 +0200
Subject: fix localstate dir

Forwarded: not-needed
---
 m4/opendnssec_common.m4 | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/m4/opendnssec_common.m4 b/m4/opendnssec_common.m4
index b1e375e..db9f56e 100644
--- a/m4/opendnssec_common.m4
+++ b/m4/opendnssec_common.m4
@@ -16,8 +16,8 @@ OPENDNSSEC_LIB_DIR=$full_libdir/opendnssec
 OPENDNSSEC_LIBEXEC_DIR=$full_libexecdir/opendnssec
 OPENDNSSEC_DATA_DIR=$full_datadir/opendnssec
 OPENDNSSEC_SYSCONF_DIR=$full_sysconfdir/opendnssec
-OPENDNSSEC_LOCALSTATE_DIR="$full_localstatedir/opendnssec"
-OPENDNSSEC_PID_DIR="$full_localstatedir/run/opendnssec"
+OPENDNSSEC_LOCALSTATE_DIR="$full_localstatedir/lib/opendnssec"
+OPENDNSSEC_PID_DIR="/run/opendnssec"
 
 AC_SUBST([OPENDNSSEC_BIN_DIR])
 AC_SUBST([OPENDNSSEC_SBIN_DIR])
