From: =?utf-8?b?T25kxZllaiBTdXLDvQ==?= <ondrej@debian.org>
Date: Wed, 21 Oct 2015 10:46:05 +0200
Subject: ods-control.in fixes

Forwarded: not-needed
---
 tools/ods-control.in | 15 +++++++++++++++
 1 file changed, 15 insertions(+)

diff --git a/tools/ods-control.in b/tools/ods-control.in
index 5f92d32..f40cd72 100644
--- a/tools/ods-control.in
+++ b/tools/ods-control.in
@@ -37,6 +37,7 @@ case "$1" in
 'hsm')
 	shift
 	"$bindir/ods-hsmutil" $@
+	exit $?
 	;;
 
 'signer')
@@ -66,9 +67,15 @@ case "$1" in
 		
 		exit $RETVAL
 		;;
+	stop)
+		echo "Stopping signer engine.."
+		"$sbindir/ods-signer" stop
+		exit $?
+		;;
 	*)
 		shift
 		"$sbindir/ods-signer" "$@"
+		exit $?
 		;;
 
 	esac
@@ -106,6 +113,7 @@ case "$1" in
 	*)
 		shift
 		"$sbindir/ods-enforcer" "$@"
+		exit $?
 		;;
 
 	esac
@@ -126,12 +134,19 @@ case "$1" in
 'stop')
 	echo "Stopping enforcer.."
 	"$0" enforcer stop
+	RETVAL_E=$?
 	echo "Stopping signer engine..."
 	"$0" signer stop
+	RETVAL_S=$?
+	if [ $RETVAL_E != 0 ]; then
+	    exit $RETVAL_E
+	fi
+	exit $RETVAL_S
 	;;
 
 *)
 	echo "usage: $progname hsm|signer|enforcer|start|stop ..."
+	exit 3
 	;;
 
 esac
