From: =?utf-8?q?Ond=C5=99ej_Sur=C3=BD?= <ondrej@debian.org>
Date: Wed, 21 Oct 2015 10:46:05 +0200
Subject: ods-control.in fixes

---
 tools/ods-control.in | 18 ++++++++++++++----
 1 file changed, 14 insertions(+), 4 deletions(-)

diff --git a/tools/ods-control.in b/tools/ods-control.in
index d5c550f..1d0838f 100644
--- a/tools/ods-control.in
+++ b/tools/ods-control.in
@@ -42,11 +42,13 @@ case "$1" in
 'ksm')
 	shift
 	"$bindir/ods-ksmutil" $@
+	exit $?
 	;;
 
 'hsm')
 	shift
 	"$bindir/ods-hsmutil" $@
+	exit $?
 	;;
 
 'signer')
@@ -86,9 +88,15 @@ case "$1" in
 
 		exit $RETVAL
 		;;
+	stop)
+		echo "Stopping signer engine.."
+		"$sbindir/ods-signer" stop
+		exit $?
+		;;
 	*)
 		shift
 		"$sbindir/ods-signer" "$@"
+		exit $?
 		;;
 
 	esac
@@ -153,7 +161,8 @@ case "$1" in
 		;;
 
 	*)
-		echo "usage: $progname enforcer start|stop|notify"
+		echo "usage: $progname enforcer start|stop|notify" >&2
+		exit 3
 		;;
 
 	esac
@@ -173,12 +182,13 @@ case "$1" in
 
 'stop')
 	"$0" enforcer stop
-	echo "Stopping signer engine..."
-	"$sbindir/ods-signer" stop
+	"$0" signer stop
+	exit $?
 	;;
 
 *)
-	echo "usage: $progname ksm|hsm|signer|enforcer|start|stop ..."
+	echo "usage: $progname ksm|hsm|signer|enforcer|start|stop ..." >&2
+	exit 3
 	;;
 
 esac
