From: =?utf-8?q?Ond=C5=99ej_Sur=C3=BD?= <ondrej@debian.org>
Date: Wed, 21 Oct 2015 10:46:05 +0200
Subject: fix manpages errors

---
 enforcer/utils/ods-ksmutil.1.in | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/enforcer/utils/ods-ksmutil.1.in b/enforcer/utils/ods-ksmutil.1.in
index 549b79b..824ee1c 100644
--- a/enforcer/utils/ods-ksmutil.1.in
+++ b/enforcer/utils/ods-ksmutil.1.in
@@ -7,28 +7,28 @@
 .LP
 .B ods\-ksmutil setup
 .br
-.B ods\-ksmutil start|stop|notify
+.B ods\-ksmutil [ start | stop | notify ]
 .br
-.B ods\-ksmutil update kasp|zonelist|conf|all
+.B ods\-ksmutil update [ kasp | zonelist | conf | all ]
 .br
-.B ods\-ksmutil zone add|delete|list
+.B ods\-ksmutil zone [ add | delete | list ]
 .RB ...
 .br
-.B ods\-ksmutil zonelist import|export
+.B ods\-ksmutil zonelist [ import | export ]
 .br
-.B ods\-ksmutil key generate|import|export|list|purge|rollover|ksk\-retire|ds\-seen|delete
+.B ods\-ksmutil key [ generate | import | export | list | purge | rollover | ksk\-retire | ds\-seen | delete ]
 .RB ...
 .br
 .B ods\-ksmutil rollover list
 .RB ...
 .br
-.B ods\-ksmutil policy export|import|purge
+.B ods\-ksmutil policy [ export | import | purge ]
 .RB ...
 .br
 .B ods\-ksmutil repository list
 .RB ...
 .br
-.B ods\-ksmutil backup list|prepare|commit|rollback|done
+.B ods\-ksmutil backup [ list | prepare | commit | rollback | done ]
 .br
 .B ods\-ksmutil database backup
 .RB ...
