From: =?utf-8?q?Ond=C5=99ej_Sur=C3=BD?= <ondrej@sury.org>
Date: Mon, 18 Apr 2016 15:21:10 +0200
Subject: Fail gracefully when no HSM repositories were found

---
 enforcer/enforcerd/enforcer.c | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/enforcer/enforcerd/enforcer.c b/enforcer/enforcerd/enforcer.c
index 6115217..166576b 100644
--- a/enforcer/enforcerd/enforcer.c
+++ b/enforcer/enforcerd/enforcer.c
@@ -147,7 +147,11 @@ server_main(DAEMONCONFIG *config)
                         log_msg(config, LOG_ERR, "hsm_open() result: %d", result);
                 }
             }
-            exit(1);
+            if (result == HSM_NO_REPOSITORIES) {
+                exit(0);
+	    } else {
+                exit(1);
+            }
         }
         log_msg(config, LOG_INFO, "HSM opened successfully.");
         ctx = hsm_create_context();
