From: =?utf-8?b?T25kxZllaiBTdXLDvQ==?= <ondrej@sury.org>
Date: Mon, 2 Jan 2017 15:00:13 +0100
Subject: Use SQLITE3_SONAME macro to dlopen libsqlite3.so.*

---
 enforcer/src/ods-migrate.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/enforcer/src/ods-migrate.c b/enforcer/src/ods-migrate.c
index aece505..5d6cc1c 100644
--- a/enforcer/src/ods-migrate.c
+++ b/enforcer/src/ods-migrate.c
@@ -97,7 +97,7 @@ dblayer_sqlite3_initialize(void)
     char const *error;
 
     dlerror();
-    handle = dlopen("libsqlite3.so", RTLD_NOW);
+    handle = dlopen(SQLITE3_SONAME, RTLD_NOW);
     if ((error = dlerror()) != NULL) {
 	    printf("Failed to load sqlite3 library. dlerror(): %s\n", error);
 	    exit(1);
