From: Mathieu Mirmont <mat@parad0x.org>
Date: Mon, 7 Jan 2019 22:13:30 +0100
Subject: Fix the ods-enforcer.8 man page

---
 enforcer/man/ods-enforcer.8.in | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/enforcer/man/ods-enforcer.8.in b/enforcer/man/ods-enforcer.8.in
index 87e1a02..1da9d97 100644
--- a/enforcer/man/ods-enforcer.8.in
+++ b/enforcer/man/ods-enforcer.8.in
@@ -12,12 +12,7 @@ help
 | running
 .br
 .B ods\-enforcer
-queue
-| flush
-| signconf
-| enforce
-| verbosity
-.R <number>
+queue | flush | signconf | enforce | verbosity <number>
 .br
 .B ods\-enforcer update 
 conf | repositorylist | all
