From 5d0dc26efd9c2138460029ca72715a482891a0e4 Mon Sep 17 00:00:00 2001
From: Mathieu Mirmont <mat@parad0x.org>
Date: Fri, 18 Jan 2019 00:28:23 +0100
Subject: Strip the build directory from the doxygen docs

---
 Doxyfile.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Doxyfile.in b/Doxyfile.in
index eca17f5e..44d51e6d 100644
--- a/Doxyfile.in
+++ b/Doxyfile.in
@@ -114,7 +114,7 @@ FULL_PATH_NAMES        = YES
 # If left blank the directory from which doxygen is run is used as the 
 # path to strip.
 
-STRIP_FROM_PATH        = 
+STRIP_FROM_PATH        = $(SRCDIR)
 
 # The STRIP_FROM_INC_PATH tag can be used to strip a user-defined part of 
 # the path mentioned in the documentation of a class, which tells 
-- 
2.20.1

