--- a/conf/conf.xml.in
+++ b/conf/conf.xml.in
@@ -6,11 +6,13 @@
 
 	<RepositoryList>
 
+<!--
 		<Repository name="SoftHSM">
 			<Module>@pkcs11_softhsm_module@</Module>
 			<TokenLabel>OpenDNSSEC</TokenLabel>
 			<PIN>1234</PIN>
 		</Repository>
+-->
 
 <!--
 		<Repository name="sca6000">
@@ -45,7 +47,7 @@
 		</Privileges>
 -->
 
-		<Datastore><SQLite>@OPENDNSSEC_STATE_DIR@/kasp.db</SQLite></Datastore>
+		<Datastore><SQLite>@OPENDNSSEC_STATE_DIR@/db/kasp.db</SQLite></Datastore>
 		<Interval>PT3600S</Interval>
 		<!-- <ManualKeyGeneration/> -->
 		
@@ -56,12 +58,10 @@
 	</Enforcer>
 
 	<Signer>
-<!--
 		<Privileges>
 			<User>opendnssec</User>
 			<Group>opendnssec</Group>
 		</Privileges>
--->
 
 		<WorkingDirectory>@OPENDNSSEC_STATE_DIR@/tmp</WorkingDirectory>
 		<WorkerThreads>8</WorkerThreads>
@@ -80,12 +80,10 @@
 	</Signer>
 
 	<Auditor>
-<!--
 		<Privileges>
 			<User>opendnssec</User>
 			<Group>opendnssec</Group>
 		</Privileges>
--->
 
 		<WorkingDirectory>@OPENDNSSEC_STATE_DIR@/tmp</WorkingDirectory>
 	</Auditor>
