--- a/conf/conf.xml.in
+++ b/conf/conf.xml.in
@@ -6,12 +6,14 @@
 
 	<RepositoryList>
 
+<!--
 		<Repository name="SoftHSM">
 			<Module>@pkcs11_softhsm_module@</Module>
 			<TokenLabel>OpenDNSSEC</TokenLabel>
 			<PIN>1234</PIN>
 			<SkipPublicKey/>
 		</Repository>
+-->
 
 <!--
 		<Repository name="sca6000">
@@ -38,14 +40,12 @@
 	</Common>
 
 	<Enforcer>
-<!--
 		<Privileges>
 			<User>opendnssec</User>
 			<Group>opendnssec</Group>
 		</Privileges>
--->
 
-		<Datastore><SQLite>@OPENDNSSEC_STATE_DIR@/kasp.db</SQLite></Datastore>
+		<Datastore><SQLite>@OPENDNSSEC_STATE_DIR@/db/kasp.db</SQLite></Datastore>
 		<Interval>PT3600S</Interval>
 		<!-- <ManualKeyGeneration/> -->
 		<!-- <RolloverNotification>P14D</RolloverNotification> -->
@@ -57,12 +57,10 @@
 	</Enforcer>
 
 	<Signer>
-<!--
 		<Privileges>
 			<User>opendnssec</User>
 			<Group>opendnssec</Group>
 		</Privileges>
--->
 
 		<WorkingDirectory>@OPENDNSSEC_STATE_DIR@/tmp</WorkingDirectory>
 		<WorkerThreads>4</WorkerThreads>
