--- a/tools/ods-control.in
+++ b/tools/ods-control.in
@@ -39,11 +39,13 @@ case "$1" in
 'ksm')
 	shift
 	"$bindir/ods-ksmutil" $@
+	exit $?
 	;;
 
 'hsm')
 	shift
 	"$bindir/ods-hsmutil" $@
+	exit $?
 	;;
 
 'signer')
@@ -70,6 +72,7 @@ case "$1" in
 		;;
 	*)
 		"$sbindir/ods-signer" $2
+		exit $?
 		;;
 
 	esac
@@ -139,7 +142,8 @@ case "$1" in
 		;;
 
 	*)
-		echo "usage: $progname enforcer start|stop|notify"
+		echo "usage: $progname enforcer start|stop|notify" >&2
+		exit 3
 		;;
 
 	esac
@@ -147,21 +151,26 @@ case "$1" in
 
 'start')
 	"$0" enforcer start
-
+	RETVAL=$?
+	if [ "$RETVAL" != "0" ]; then
+	    exit $RETVAL
+	fi
 	echo "Starting signer engine..."
 	"$0" signer start
-
+	exit $?
 	;;
 
 'stop')
 	"$0" enforcer stop
 
 	echo "Stopping signer engine.."
-	"$sbindir/ods-signer" stop
+	"$0" signer stop
+	exit $?
 	;;
 
 *)
-	echo "usage: $progname ksm|hsm|signer|enforcer|start|stop ..."
+	echo "usage: $progname ksm|hsm|signer|enforcer|start|stop ..." >&2
+	exit 3
 	;;
 
 esac
