--- opendnssec.orig/tools/ods-control.in
+++ opendnssec/tools/ods-control.in
@@ -40,11 +40,13 @@ case "$1" in
 'ksm')
 	shift
 	"$bindir/ods-ksmutil" $@
+	exit $?
 	;;
 
 'hsm')
 	shift
 	"$bindir/ods-hsmutil" $@
+	exit $?
 	;;
 
 'signer')
@@ -84,9 +86,15 @@ case "$1" in
 		
 		exit $RETVAL
 		;;
+	stop)
+		echo "Stopping signer engine.."
+		"$sbindir/ods-signer" stop
+		exit $?
+		;;
 	*)
 		shift
 		"$sbindir/ods-signer" "$@"
+		exit $?
 		;;
 
 	esac
@@ -151,7 +159,8 @@ case "$1" in
 		;;
 
 	*)
-		echo "usage: $progname enforcer start|stop|notify"
+		echo "usage: $progname enforcer start|stop|notify" >&2
+		exit 3
 		;;
 
 	esac
@@ -171,12 +180,13 @@ case "$1" in
 
 'stop')
 	"$0" enforcer stop
-	echo "Stopping signer engine..."
-	"$sbindir/ods-signer" stop
+	"$0" signer stop
+	exit $?
 	;;
 
 *)
-	echo "usage: $progname ksm|hsm|signer|enforcer|start|stop ..."
+	echo "usage: $progname ksm|hsm|signer|enforcer|start|stop ..." >&2
+	exit 3
 	;;
 
 esac
