--- a/enforcer/utils/ods-ksmutil.1.in
+++ b/enforcer/utils/ods-ksmutil.1.in
@@ -1,4 +1,4 @@
-.TH "ods-ksmutil" "1" "February 2010" OpenDNSSEC "OpenDNSSEC ods-ksmutil"
+.TH "ods-ksmutil" "1" "2012-10-24" OpenDNSSEC "OpenDNSSEC ods-ksmutil"
 .\" $Id: ods-ksmutil.1.in 6768 2012-10-24 11:15:35Z sion $
 .SH "NAME"
 .LP
@@ -8,31 +8,29 @@
 .LP
 .B ods-ksmutil setup
 .br
-.B ods-ksmutil start|stop|notify
+.B ods-ksmutil [ start | stop | notify ]
 .br
-.B ods-ksmutil update kasp|zonelist|conf|all
+.B ods-ksmutil update [ kasp | zonelist | conf | all ]
 .br
-.B ods-ksmutil zone add|delete|list
+.B ods-ksmutil zone [ add | delete | list ]
 .RB ...
 .br
-.B ods-ksmutil zonelist import|export
+.B ods-ksmutil zonelist [ import | export ]
 .br
-.B ods-ksmutil key 
-.RS
-.B		generate|import|export|list|purge|rollover|ksk\-retire|ds\-seen|delete
+.B ods-ksmutil key [ generate | import | export | list | purge | rollover | ksk\-retire | ds\-seen | delete ]
 .RB ...
 .RE
 .br
 .B ods-ksmutil rollover list
 .RB ...
 .br
-.B ods-ksmutil policy export|import|purge
+.B ods-ksmutil policy [ export | import | purge ]
 .RB ...
 .br
 .B ods-ksmutil repository list
 .RB ...
 .br
-.B ods-ksmutil backup list|prepare|commit|rollback|done
+.B ods-ksmutil backup [ list | prepare | commit | rollback | done ]
 .br
 .B ods-ksmutil database backup
 .RB ...
--- a/signer/man/ods-signer.8.in
+++ b/signer/man/ods-signer.8.in
@@ -1,4 +1,4 @@
-.TH "ods-signer" "22" "February 2012" "OpenDNSSEC" "OpenDNSSEC ods-signer"
+.TH "ods-signer" "8" "2010-08-10" "OpenDNSSEC" "OpenDNSSEC ods-signer"
 .\" $Id: ods-signer.8.in 3709 2010-08-10 13:03:45Z rb $
 .SH "NAME"
 .B ods\-signer 
--- a/signer/man/ods-signerd.8.in
+++ b/signer/man/ods-signerd.8.in
@@ -1,4 +1,4 @@
-.TH "ods-signerd" "22" "February 2012" "OpenDNSSEC" "OpenDNSSEC ods-signerd"
+.TH "ods-signerd" "8" "2010-02-02" "OpenDNSSEC" "OpenDNSSEC ods-signerd"
 .\" $Id: ods-signerd.8.in 2785 2010-02-02 16:45:00Z rb $
 .SH "NAME"
 .LP
--- a/enforcer/enforcerd/ods-enforcerd.8.in
+++ b/enforcer/enforcerd/ods-enforcerd.8.in
@@ -1,4 +1,4 @@
-.TH "ods-enforcerd" "8" "February 2010" "OpenDNSSEC" "OpenDNSSEC ods-enforcerd"
+.TH "ods-enforcerd" "8" "2012-01-04" "OpenDNSSEC" "OpenDNSSEC ods-enforcerd"
 .\" $Id: ods-enforcerd.8.in 6013 2012-01-04 15:57:01Z sion $
 .SH "NAME"
 .LP
--- a/enforcer/utils/ods-kaspcheck.1.in
+++ b/enforcer/utils/ods-kaspcheck.1.in
@@ -1,4 +1,4 @@
-.TH "ods-kaspcheck" "1" "February 2010" "OpenDNSSEC" "OpenDNSSEC ods-kaspcheck"
+.TH "ods-kaspcheck" "1" "2012-01-04" "OpenDNSSEC" "OpenDNSSEC ods-kaspcheck"
 .\" $Id: ods-kaspcheck.1.in 6014 2012-01-04 16:29:23Z sion $
 .SH "NAME"
 .LP
--- a/libhsm/src/bin/ods-hsmspeed.1.in
+++ b/libhsm/src/bin/ods-hsmspeed.1.in
@@ -1,4 +1,4 @@
-.TH "ods-hsmspeed" "1" "February 2010" "OpenDNSSEC" "OpenDNSSEC ods-hsmspeed"
+.TH "ods-hsmspeed" "1" "2012-08-28" "OpenDNSSEC" "OpenDNSSEC ods-hsmspeed"
 .\" $Id: ods-hsmspeed.1.in 6560 2012-08-28 06:31:40Z rb $
 .SH "NAME"
 .LP
--- a/libhsm/src/bin/ods-hsmutil.1.in
+++ b/libhsm/src/bin/ods-hsmutil.1.in
@@ -1,4 +1,4 @@
-.TH "ods-hsmutil" "1" "February 2010" "OpenDNSSEC" "OpenDNSSEC ods-hsmutil"
+.TH "ods-hsmutil" "1" "2012-09-20" "OpenDNSSEC" "OpenDNSSEC ods-hsmutil"
 .\" $Id: ods-hsmutil.1.in 6704 2012-09-20 10:43:19Z rb $
 .SH "NAME"
 .LP
--- a/tools/ods-control.8.in
+++ b/tools/ods-control.8.in
@@ -1,4 +1,4 @@
-.TH "ods-control" "8" "February 2010" "OpenDNSSEC" "OpenDNSSEC ods-control"
+.TH "ods-control" "8" "2012-01-04" "OpenDNSSEC" "OpenDNSSEC ods-control"
 .\" $Id: ods-control.8.in 6011 2012-01-04 15:24:18Z jakob $
 .SH "NAME"
 .LP
