--- a/conf/conf.xml.in
+++ b/conf/conf.xml.in
@@ -1,7 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<!-- $Id: conf.xml.in 4998 2011-04-21 12:29:27Z jakob $ -->
-
 <Configuration>
 
 	<RepositoryList>
--- a/conf/kasp.xml.in
+++ b/conf/kasp.xml.in
@@ -1,7 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<!-- $Id: kasp.xml.in 4998 2011-04-21 12:29:27Z jakob $ -->
-
 <!--
   
   NOTE:  The default policy below is a TEMPLATE ONLY and should be reviewed
--- a/conf/signconf.xml.in
+++ b/conf/signconf.xml.in
@@ -1,7 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<!-- $Id: signconf.xml.in 4998 2011-04-21 12:29:27Z jakob $ -->
-
 <SignerConfiguration>
 	<Zone name="opendnssec.org">
 
--- a/conf/zonefetch.xml.in
+++ b/conf/zonefetch.xml.in
@@ -1,7 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<!-- $Id: zonefetch.xml.in 4998 2011-04-21 12:29:27Z jakob $ -->
-
 <ZoneFetch>
 	<!-- where to listen for notifies -->
 	<!-- DEFAULT: do not listen to notify on specific address -->
--- a/conf/zonelist.xml.in
+++ b/conf/zonelist.xml.in
@@ -1,7 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<!-- $Id: zonelist.xml.in 4998 2011-04-21 12:29:27Z jakob $ -->
-
 <ZoneList>
 <!--
 	<Zone name="example.com">
