--- a/enforcer/utils/database_create.sqlite3
+++ b/enforcer/utils/database_create.sqlite3
@@ -114,8 +114,6 @@ create table dnsseckeys (
   retire        varchar(64) null default null,  -- time when the key retires
   dead          varchar(64) null default null,  -- time when key is slated for removal
 
-
-  foreign key (zone_id) references zones (id),
   foreign key (keypair_id) references keypairs (id)
 );
 
--- a/enforcer/utils/database_create.mysql
+++ b/enforcer/utils/database_create.mysql
@@ -125,8 +125,7 @@ create table dnsseckeys (
   dead          timestamp null default null,  # time when key is slated for removal
 
   constraint primary key (id),
-  constraint foreign key (zone_id) references zones (id),
-  constraint foreign key (keypair_id) references keypairs (id)
+  constraint dnsseckeys_keypairs_id foreign key (keypair_id) references keypairs (id)
 )ENGINE=InnoDB;
 
 # parameters_policies - join table to hold the values of parameters
